app/assets/stylesheets/color_themes/_color_theme_override_origwhite.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  #main_stream .stream_element {
+    border: 0;
+    border-bottom: 1px solid $border-grey;
+    margin-bottom: 10px;
+  }
+
+  .profile_header > div {
+    border-bottom: 1px solid $border-grey;
+  }
+
+  .left-navbar .hoverable { border-bottom: 0; }
+
+  .profile-sidebar,
+  .left-navbar-fixed-background,
+  &.page-tags .left-navbar-fixed-background,
+  .left-navbar {
+    border-right: 1px solid $border-grey;
```

Comment:
You should put a border-left on the content instead. With that, the border stop at the end of the left elem which is usually pretty short (look at a profile page with only an avatar for example).